packages/flutter_tools/test/web.shard/hot_reload_web_errors_test.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+@TestOn('linux') // https://github.com/flutter/flutter/issues/169304
```

Comment:
This file, and the ones below, have been running on all platforms for a while and not causing timeouts. Should we keep them that way if there's value in running them on different platforms?

Reply:
I would prefer to run more tests on all platforms but my understanding from the discussion on https://github.com/flutter/flutter/issues/169304 was that the timeouts were increased for the mac and windows configurations and they also have the least amount of resources available. To find a middle ground during this period where we have two separate modes to test I thought we could avoid running tests of the non-default settings on all platforms.